BMarcelus/VotosLocos.js
# converted site to a pwa to install and play offline

how to test:
1. go to the url
2. notice icon on the url address bar that says install
3. click install
4. pin to task
5. allow to finish install
6. now turn off wifi
7. press ctl+R to refresh the page, notice the game still loads and plays offline
8. this means it worked!
9. Don't forget to turn back on ur wifi to not have other issues lol
